source/elements/oneDPL/source/parallel_api/execution_policies.rst

```diff
@@ -180,5 +178,54 @@ as the template argument, otherwise unspecified.
 Return a policy object constructed from ``policy``, with a new kernel name provided as the template
 argument. If no policy object is provided, the new policy is constructed from ``dpcpp_default``.
 
+.. _exec-policy-type-trait:
+
+Execution Policy Type Trait
++++++++++++++++++++++++++++
+
+oneDPL provides type trait utilities to detect its execution policy types at compile time for the purpose of
+function overload resolution:
+
+.. code:: cpp
+
+  // Defined in <oneapi/dpl/execution>
+
+  namespace oneapi {
+    namespace dpl {
+
+      template <class T>
+      struct is_execution_policy { /*see below*/ };
+
+      template <class T>
+      constexpr inline bool is_execution_policy_v = oneapi::dpl::is_execution_policy<T>::value;
```

Comment:
```suggestion
      inline constexpr bool is_execution_policy_v = oneapi::dpl::is_execution_policy<T>::value;
```

I prefer that order because this is the way it's defined in C++ standard. I don't want to hold this PR because of that issue but if you decide to change it the way I am suggesting, please ping me and I approve this patch once again.

Reply:
No problem, I will do it. No re-approval is needed, thanks.